queries/typescript/locals.scm

```diff
@@ -1,2 +1,13 @@
+; Scopes
+;-------
+
+(method_definition) @scope
+(if_statement) @scope
+(for_statement) @scope
+(try_statement) @scope
+
```

Comment:
Isn't this already included in the is queries?

Reply:
I added this scope because it wasn't highlighted in a particular statement
But if this is not the correct behavior, I'm roll it back.

Should I roll back?

Reply:
All those things also exist in JS. So you can add them to the JS queries and have them in typescript automatically.
It's probably good to add `statement_block` as scope since curly braces introduce a scope. 

@steelsojka  could you review please?

Reply:
Is that so.
Thank you for teaching me
I rolled back the scope query, so check again

Reply:
Is that so.
Thank you for teaching me
I rolled back the scope query, so check again

Reply:
why not add them to JS instead?